main.py

```diff
@@ -1,13 +1,102 @@
 #!/usr/bin/env python
+import re
+from nltk.wsd import lesk
 
-# Your classes or methods go here:
 
+class JsonParsedWriter:
 
-class Tool:
-    def my_method(self):
-        print("my_method called!")
+    @staticmethod
+    def write(data, output_name, indent=2):
```

Comment:
1. `out_filename` could be a better parameter name than `output_name`.
2. It's a good idea to provide a default value for the `indent` param.